README.md

```diff
@@ -98,43 +42,45 @@ And just like that you're started!
 Then make your first FlowRepresentable view:
 ```swift
 import SwiftCurrent
+import SwiftUI
 struct OptionalView: View, FlowRepresentable {
     weak var _workflowPointer: AnyFlowRepresentable?
     let input: String
     init(with args: String) { input = args }
     var body: some View { Text("Only shows up if no input") }
     func shouldLoad() -> Bool { input.isEmpty }
 }
-struct ExampleView: View, FlowRepresentable {
+struct ExampleView: View, PassthroughFlowRepresentable {
     weak var _workflowPointer: AnyFlowRepresentable?
     var body: some View { Text("This is ExampleView!") }
 }
 ```
-Then from your ContentView body, add: 
+Then from your `ContentView` or whatever view (or app) you'd like to contain the workflow, add the following view to the body: 
 ```swift
 import SwiftCurrent_SwiftUI
-...
-WorkflowLauncher(isLaunched: .constant(true), startingArgs: "Skip optional screen") {
-    thenProceed(with: OptionalView.self) {
-        thenProceed(with: ExampleView.self)
+// ...
+var body: some View { 
+    // ... other view code (if any)
+    WorkflowLauncher(isLaunched: .constant(true), startingArgs: "Skip optional screen") {
+        thenProceed(with: OptionalView.self) {
+            thenProceed(with: ExampleView.self)
+        }
     }
 }
 ```
 
-And just like that you're started!
-
-Here's an example of an app written with SwiftCurrent running wherever SwiftUI runs.  Check it out:
-
-https://user-images.githubusercontent.com/79471462/131555558-9e01e753-cafd-4ae2-af1b-b71a1aabb71f.mp4
-
+And just like that you've got a workflow! You can now add more items to it, or re-order the items that are there. To understand more of how this works and what is there [check out our developer docs](https://wwt.github.io/SwiftCurrent/How%20to%20use%20SwiftCurrent%20with%20SwiftUI.html)
 
-# Deep Dive
+# Check out our example apps
+We have [example apps](https://github.com/wwt/SwiftCurrent/tree/main/ExampleApps) for both SwiftUI and UIKit that show SwiftCurrent in action. They're even tested so you can see what it's like to test SwiftCurrent code. To run it locally, start by cloning the repo, open `SwiftCurrent.xcworkspace` and then run the `SwiftUIExample` scheme or the `UIKitExample` scheme. 
 
+# [See this to learn more](https://wwt.github.io/SwiftCurrent/index.html)
+For specific documentation check out:
 - [Why SwiftCurrent?](https://wwt.github.io/SwiftCurrent/why-this-library.html)
 - [Installation](https://wwt.github.io/SwiftCurrent/installation.html)
```

Comment:
This installation file also needs to be updated because it says BETA.

Reply:
Should that wait for the PR that renames it? Or you wanna just do it here?

Reply:
🤷  I'm indifferent. As long as we don't forget.

Reply:
We won't, it's on the list.